api/v1alpha1/orchestrator_types.go

```diff
@@ -44,7 +44,7 @@ type OrchestratorSpec struct {
 	PostgresDB Postgres `json:"postgres"`
 
 	// Configuration for Orchestrator. Optional
-	OrchestratorConfig OrchestratorConfig `json:"orchestrator,omitempty"`
+	ServerlessWorkflow ServerlessWorkflow `json:"orchestrator,omitempty"`
```

Comment:
You forgot to update `json:"orchestrator,omitempty"` :)

Reply:
This is related to the confusing field name. So I'm thinking to hold off on making the change and see what the team says.